samples/install-without-bom/pom.xml

```diff
@@ -144,7 +145,7 @@
           <systemPropertyVariables>
             <spanner.test.instance>java-client-integration-test</spanner.test.instance>
             <spanner.test.instance.mr>java-client-mr-integration-test</spanner.test.instance.mr>
-            <spanner.test.instance.config>nam15</spanner.test.instance.config>
+            <spanner.test.instance.config>nam11</spanner.test.instance.config>
```

Comment:
This change was needed because `nam15` does not have any optional replicas. And because of this existing integration tests were broken for `CreateInstanceConfigSample` class. Particularly below line
```
List<ReplicaInfo> readOnlyReplicas =
          ImmutableList.of(baseConfig.getOptionalReplicas().get(0));
```

I have modified this config and that fixes existing broken test.